src/pages/individual/Individual.jsx

```diff
@@ -143,7 +149,7 @@ export default function Individual() {
     );
     const firstNineAssets = slice(uniqueModifiedAssets, 0, 9);
     return firstNineAssets;
-  }, [individualData]);
+  }, [individualData, intl]);
```

Comment:
I hadn't thought that intl would ever change?

Reply:
(maybe this is a lifecycle question ultimately).

Reply:
If we ever allow language switching in the application, it will.